src/assets/fonts/style.scss

```diff
@@ -0,0 +1,90 @@
+@font-face {
+  font-family: 'icomoon';
+  src:  url('../../assets/fonts/icomoon.eot?7ctv17');
+  src:  url('../../assets/fonts/icomoon.eot?7ctv17#iefix') format('embedded-opentype'),
+    url('../../assets/fonts/icomoon.ttf?7ctv17') format('truetype'),
+    url('../../assets/fonts/icomoon.woff?7ctv17') format('woff'),
+    url('../../assets/fonts/icomoon.svg?7ctv17#icomoon') format('svg');
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
+
+[class^="icon-"], [class*=" icon-"] {
+  /* use !important to prevent issues with browser extensions that change fonts */
+  font-family: 'icomoon' !important;
+  speak: never;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+
+  /* Better Font Rendering =========== */
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.icon-stopwatch:before {
+  content: "\e952";
+}
+.icon-user-tie:before {
+  content: "\e976";
+}
+.icon-spinner:before {
+  content: "\e97a";
+}
+.icon-glass2:before {
+  content: "\e9a1";
+}
+.icon-spoon-knife:before {
+  content: "\e9a3";
+}
+.icon-leaf:before {
+  content: "\e9a4";
+}
+.icon-eye:before {
+  content: "\e9ce";
+}
+.icon-eye-blocked:before {
+  content: "\e9d1";
+}
+.icon-bookmark:before {
+  content: "\e9d2";
+}
+.icon-bookmarks:before {
+  content: "\e9d3";
+}
+.icon-star-empty:before {
```

Comment:
this is the way to turn icons into fonts, so that they can be easier manipulated. All icons should be in one file and then they are used where needed. In my case it's src/assets/fonts/style.scss.